### PR TITLE
Updating Reload Scrapers formatting

### PR DESCRIPTION
**Pull request by Gykes**
Per convo with people on Discord. I have updated the Reload Scrapers UI. It now adds a button if the filter box appears and then the button extends and takes up the whole space if the filter box does not exist. Added in different `<div>`s as well so that it was easier to target for theming/plugins

The CSS is... janky at best. Not too great with it so opinions and potential fixes would be greatly appreciated. I figured using the default colors was best practice and people can theme them to be whatever they want.

With filter: 
filter

Without filter
no filter

closes https://github.com/stashapp/stash/issues/5233

### CODE REVIEW

**Review by cj12312021 — COMMENTED** [review]
I'll be pushing changes shortly.

**Comment by cj12312021** [comment]
I've wrapped up my changes. I'm just waiting to push the assuming you are okay with me pushing directly to your branch. I did end up redoing much of what was there due to UI issues, which I couldn't pinpoint the source of. But the core concept we discussed yesterday is still intact. For your awareness, I left comments in the code to call your attention to changes that could have contributed to the issue I was seeing or changes that were not necessary. The hope is that this will help you avoid some of these pitfalls as you continue your work here.

The video below shows the main issues I was seeing. A decent portion of the drop-down menu fell outside of the edit tab, leading to those sections being cut off, and the button group at the top did not correctly fill the width of the menu. Then, there was the centering issue I mentioned with the Reload Scrapers label and the SVG it used.

https://github.com/user-attachments/assets/8ef522df-9f65-415a-b8ed-8b8397a23193

### The Changes
I've attached a video here showing the updates I made to the section. The abovementioned issues have been addressed. The button group now stays in view when scrolling through scrapers. There was a discussion yesterday about an appropriate style for the scrapper button to address the contrast issue "issue" I was seeing with that button and the Scrape/search buttons above it. Ultimately, I realized that a `Reload Scrapers` button already exists in Stash in the Settings > Metadata Providers > Scrapers section, so it was most appropriate to leave the button design the same in the source for consistency. However, I adjusted the search box style, which helped counter some of the visual issues I had with the combination.

https://github.com/user-attachments/assets/33605d3b-9b49-4683-bf76-b5c82d9e78e9

Screenshot 2024-09-08 131720

**Comment by Gykes** [comment]
> I've wrapped up my changes. I'm just waiting to push the assuming you are okay with me pushing directly to your branch. I did end up redoing much of what was there due to UI issues, which I couldn't pinpoint the source of. But the core concept we discussed yesterday is still intact. For your awareness, I left comments in the code to call your attention to changes that could have contributed to the issue I was seeing or changes that were not necessary. The hope is that this will help you avoid some of these pitfalls as you continue your work here.
> 
> The video below shows the main issues I was seeing. A decent portion of the drop-down menu fell outside of the edit tab, leading to those sections being cut off, and the button group at the top did not correctly fill the width of the menu. Then, there was the centering issue I mentioned with the Reload Scrapers label and the SVG it used.
> 
>  Screen.Recording.2024-09-08.124456.mp4 
> ### The Changes
> I've attached a video here showing the updates I made to the section. The abovementioned issues have been addressed. The button group now stays in view when scrolling through scrapers. There was a discussion yesterday about an appropriate style for the scrapper button to address the contrast issue "issue" I was seeing with that button and the Scrape/search buttons above it. Ultimately, I realized that a `Reload Scrapers` button already exists in Stash in the Settings > Metadata Providers > Scrapers section, so it was most appropriate to leave the button design the same in the source for consistency. However, I adjusted the search box style, which helped counter some of the visual issues I had with the combination.
> 
>  Screen.Recording.2024-09-08.122954.mp4 
> Screenshot 2024-09-08 131720

Thanks for checking me out. I could have spent more time fully testing it but figured I would get it up and yall can tell me what's wrong... which is what happened. I'll give you write access here now once I figure it out. Thanks for the changes!

**Comment by cj12312021** [comment]
No problem! Yeah, banging away at work yourself to learn can be insightful, but after enough time, it becomes more advantageous to get assistance if it is available. 

If you go to the settings page in your fork, at the top, there will be a collaborators option. Click that option and click the green Add people button in the center. You can then provide my GitHub user name in the text box that appears and just give with `Write access`

**Comment by WithoutPants** [comment]
If we're using up the vertical space regardless, it probably doesn't make sense to hide the filter field.

**Comment by cj12312021** [comment]
I'm in favor of that mostly because the full-width button isn't very appealing.

**Comment by Gykes** [comment]
I agree, as well. I can modify it a little later. Have dinner plans tonight so it will be tomorrow that I can get to it. Feel free to do it though, CJ.

**Review by WithoutPants — APPROVED** [review]
Thanks for sorting this. 

I made some minor tweaks to the styling. The `:has` directive was no longer necessary if we always show the filter and that directive is a fairly recent addition so may not have full support.

I changed the colours to use the theme variables instead and changed the background colour of the filter input field to be `$textfield-bg` instead of a specific colour that I couldn't find elsewhere in the styling code. 

image

If that colour is actually necessary, then we should add it to the theme with an appropriate name - that can be a separate change though.

Everything else looks good.

File: ui/v2.5/src/components/Shared/styles.scss [hunk 1]
```diff
@@ -640,34 +640,31 @@ button.btn.favorite-button {
...
 
     .btn-group {
-      border: solid 1px rgba(16, 22, 26, 0.3);
+      border: solid 1px $textfield-bg;
       border-radius: 5px;
       width: 100%;
     }
 
     .clearable-text-field {
-      background-color: #26333d;
```
Comment (cj12312021) [inline ui/v2.5/src/components/Shared/styles.scss]: Good catch. I picked up the color by using the color picker tool on one of the text fields on the page. At the time, it didn't cross my mind that the opacity applied to those fields would not be applied to the color code generated by the color picker.

**Pull request merged**